Ship_Game/GameScreens/ShipDesign/ShipDesignScreen.cs

```diff
@@ -640,5 +644,18 @@ void RefreshHullSelectList()
                 }
             }
         }
+
+        public void ResetMusic()

```

Comment:
Also, there is another function called ResetMusic() in MainMenuScreen, i dont think there is a need to duplicate the code there. Just use the main function and add what you need there, maybe

Reply:
they are sligthtly different, but I can see how to unify them a bit

Reply:
Ok, so try to solve that and i'll meet you again in the review :)